### PR TITLE
Fix bridge compilation + warning

### PR DESCRIPTION
**Pull request by nzlz**
Not providing a ROS2 version for the `control_msgs` package can cause the compiler to build the ROS1 version for the ROS2 section. Not sure why this happens, and should not happen in any way, but it is happening in my system. Providing the version 2.0.0 forces the compiler to find for a ROS2 compatible version (2.0.0 is the first ROS2 build of control_msgs). In my system for instance, I have the latest 2.1.0 and the compilation succeeds, meaning this won't cause issues with future versions of the pkg.

Error solved by this PR can be found in [this gist](https://gist.github.com/nzlz/9945810e85302f41a2c1553ec5e93a2f).

Also removed a compilation warning.

### CODE REVIEW

File: individual_trajectories_bridge/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_action REQUIRED)
 find_package(rmw_implementation_cmake REQUIRED)
-find_package(control_msgs REQUIRED)
+find_package(control_msgs 2.0.0 REQUIRED)
```
Comment (rkojcev) [inline individual_trajectories_bridge/CMakeLists.txt]: I am not a fan of defining versions of packages within the CMake meaning we need to remember this and maintain if new version is release. The weird thing is that is isolated case as far as I know. We tested without setting the version on 3-4 different machines and did not produce any problem.

File: individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp [hunk 2]
```diff
@@ -11,7 +11,7 @@ FollowJointTrajectoryAction::FollowJointTrajectoryAction(std::string name, ros::
   sub_action_cancel_ = nh.subscribe( action_name_ + std::string("/cancel"), 1, &FollowJointTrajectoryAction::actionCancelCallback, this);
 }
 
-void FollowJointTrajectoryAction::actionCancelCallback(const actionlib_msgs::GoalID::ConstPtr& msg)
+void FollowJointTrajectoryAction::actionCancelCallback(const actionlib_msgs::GoalID::ConstPtr&)
```
Comment (rkojcev) [inline individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp]: Did you test if only this fix is enough without modifying the CMake?

File: individual_trajectories_bridge/CMakeLists.txt (hunk 1 above)
Comment (LanderU) [reply individual_trajectories_bridge/CMakeLists.txt]: I do agree with you. Are you using a clean machine? You can use the Docker image available for CI to test it out.

Comment (rkojcev) [reply individual_trajectories_bridge/CMakeLists.txt]: @alesolano tested this without the setting the version in a docker and he did not have any issues. @alesolano can you confirm this?

Comment (alesolano) [reply individual_trajectories_bridge/CMakeLists.txt]: Yes. Tested a couple of times, last time being yesterday.

Comment (LanderU) [reply individual_trajectories_bridge/CMakeLists.txt]: So this is not needed, I'm going to close it.

Comment (nzlz) [reply individual_trajectories_bridge/CMakeLists.txt]: Although its true that we should avoid this, this does not restrict the version. It only ensures that the compiler is capturing the ros2 version of control msgs.

If we are not confortable with this we could simply add a troubleshoot note to the readme. I think this same issue will happen in more systems, not inly mine.

File: individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp (hunk 2 above)
Comment (nzlz) [reply individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp]: This is only to hide a warning during compilation. Cmake was enough to fix the issue yes.

File: individual_trajectories_bridge/CMakeLists.txt (hunk 1 above)
Comment (rkojcev) [reply individual_trajectories_bridge/CMakeLists.txt]: Lets add troubleshoot note is easier.

File: individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp (hunk 2 above)
Comment (LanderU) [reply individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp]: Could you please create a new PR solving only the warning?

Comment (rkojcev) [reply individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp]: Yep that is a good catch. Lets make a PR to fix this warning.

**Pull request closed without merging**